.flake8

```diff
@@ -51,6 +51,11 @@ extend-ignore =
   D205,
   D400,
   D401,
+  DUO116, # use of "shell=True" is insecure in "subprocess" module
```

Comment:
Where are these coming from? Clearly not from from default set of plugins.

Reply:
don't know. About the shell=True thing, I've a patch somewhere for that and I plan to submit it once the CI is fine.

Reply:
https://github.com/dlint-py/dlint/blob/master/docs/linters/DUO116.md

i'm using https://github.com/konstruktoid/action-pylint/blob/master/requirements.txt, but if @apatard got a future patch for it why not leave it in untill merged?

Reply:
shell removal pushed as PR112.